Podfile.lock

```diff
@@ -14,6 +14,7 @@ PODS:
   - AFNetworking/Serialization (4.0.1)
   - AFNetworking/UIKit (4.0.1):
     - AFNetworking/NSURLSession
+  - DeviceKit (4.7.0)
```

Comment:
This is the best option I found for converting Apple's device identifiers into human readable strings. It's MIT, relatively small, well-maintained and somewhat popular so I hope this is an ok choice.